pkg/virt-operator/resource/generate/components/validations_generated.go

```diff
@@ -1049,6 +1049,22 @@ var CRDsValidation map[string]string = map[string]string{
               description: PullPolicy describes a policy for if/when to pull a container
                 image
               type: string
+            instancetype:
+              description: Instancetype configuration
+              nullable: true
+              properties:
+                referencePolicy:
+                  description: |-
+                    ReferencePolicy defines how an instance type or preference should be referenced by the VM after submission, supported values are:
+                    reference (default) - Where a copy of the original object is stashed in a ControllerRevision and referenced by the VM.
+                    expand - Where the instance type or preference are expanded into the VM during submission with references removed.
+                  enum:
+                  - reference
+                  - expand
+                  - expandAll
```

Comment:
Do we need to add this, if we do not support this policy?

Reply:
I'm offering a few choices at the moment, I'm not deciding which if any will be used downstream so yeah I'd like to include all three.